EFTools.msbuild

```diff
@@ -69,13 +69,11 @@
               LogStandardErrorAsError="true" />
     </Target>
 
-    <Import Project="tools\SkipStrongNames.targets" />
```

Comment:
Do we know if the removal of `SkipStrongNames.targets` might regress anything or affect the build process negatively?

Reply:
I actually saw in the build logs that it was redundant, experimentally removed, and found everything to be fine.